### ISSUE TITLE
jspm install only downloads handlebars.js

### ISSUE THREAD

**Issue by yoavniran** [components/handlebars.js#18]
Im using JSPM so I install using: "jspm install github:components/handlebars.js"

When I do this, the only file in the components folder is handlebars.js. I dont get any of the minified versions and not the handlebars.runtime which is the one I need.

Is there a way to get the runtime using this shim?

for now looks like I need to get the full release from: wycats/handlebars.js

thanks.

**Comment by yoavniran** [comment]
actually, getting the code from wycats/handlebars.js on GH doesnt help me because it doesnt have the dist files and I dont want to build handlebars in order to use it...

**Comment by kpdecker** [comment]
I'm glad to make any changes or document how to make this work under jspm, but we need someone who actually uses this to help out with that.

**Comment by yoavniran** [comment]
hi @kpdecker, Id be happy to help.
Basically, if you can include both handlebars.js and handlebars.runtime.js in the package it would be great.
Thanks!

**Comment by kpdecker** [comment]
The runtime file is there: https://github.com/components/handlebars.js/blob/master/handlebars.runtime.js

Note that we actively do not include dist files in the @wycats/handlebars.js repo as including compiled files in source make things chaotic for merges, etc. This repository is intended to be the analog to the dist/ dir and we updated it on releases.

**Comment by yoavniran** [comment]
I see the runtime js file in the repo you mentioned. But... when I do: **jspm install handlebars** I dont get it which is the problem since I need it to be there when other developers run **jspm install** or when our build system does.

**Comment by kpdecker** [comment]
Which ties into my first comment. It sounds like there is some config that needs to happen for jspm to include the additional files. Someone needs to research that, test it, and submit the changes so we can include them in the package.

**Comment by yoavniran** [comment]
got it. Will do.

**Comment by yoavniran** [comment]
see #19

**Comment by kpdecker** [comment]
Closing in favor of #19

**Comment by yoavniran** [comment]
any chance you can create a tag for this commit so i can start using it without having to point to the master? or do you only tag when upgrading to a handlebars version?

thanks

**Comment by kpdecker** [comment]
Yes, the tag will have to wait until the next handlebars release. There are a few things that need to be done before that can happen. I'm back from an extended vacation so hopefully I can get this out relatively soon.
